src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -226,13 +226,13 @@ class SourceContainer(abc.ABC):
 
     @abc.abstractmethod
     def build_dependency_graph(self, parent: DependencyGraph) -> list[DependencyProblem]:
-        raise NotImplementedError()
+        """abstract method"""
```

Comment:
Could provide a more descriptive docstring for this method?

Reply:
Same for the other abstract methods

Reply:
The purpose is to improve test coverage, not to document the method

Reply:
I understand it also increase test coverage, still a useful docstring does not hurt

Reply:
done